<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-43CROFND" name="com.employeeapi.testCases.TC002_GET_Single_Employee_Details" tests="9" failures="2" timestamp="2020-03-26T14:45:42 IST" time="0.047" errors="0">
  <testcase name="checkContentEncoding" time="0.005" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkCookies" time="0.004" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkServerType" time="0.004" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkStatusCode" time="0.010" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkStatusLine" time="0.004" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkResponseBody" time="0.008" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details">
    <failure type="java.lang.AssertionError" message="expected:&lt;true&gt; but was:&lt;false&gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:147)
at com.employeeapi.testCases.TC002_GET_Single_Employee_Details.checkResponseBody(TC002_GET_Single_Employee_Details.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkResponseBody -->
  <system-out/>
  <testcase name="checkResponseTime" time="0.000" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkContentlength" time="0.003" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details"/>
  <system-out/>
  <testcase name="checkContentType" time="0.009" classname="com.employeeapi.testCases.TC002_GET_Single_Employee_Details">
    <failure type="org.junit.ComparisonFailure" message="expected:&lt;...xt/html; charset=UTF[]]-8&gt; but was:&lt;...xt/html; charset=UTF[]-8&gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<...xt/html; charset=UTF[]]-8> but was:<...xt/html; charset=UTF[]-8>
at org.junit.Assert.assertEquals(Assert.java:125)
at org.junit.Assert.assertEquals(Assert.java:147)
at com.employeeapi.testCases.TC002_GET_Single_Employee_Details.checkContentType(TC002_GET_Single_Employee_Details.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <system-out/>
</testsuite> <!-- com.employeeapi.testCases.TC002_GET_Single_Employee_Details -->
